CHANGELOG.md

```diff
@@ -98,9 +98,10 @@ All notable changes to this project will be documented in this file, per [the Ke
 ## [1.0.9] - 2022-02-18
 ### Added
 - `hostname` and `path` fields to schema (props [@Rahmon](https://github.com/Rahmon), [@dinhtungdu](https://github.com/dinhtungdu) via [#164](https://github.com/globeandmail/sophi-for-wordpress/pull/164)).
+- `found_posts` filter to modify the Sophi API response to allow manual posts to be injected into the response (e.g., via a fallback method to inject posts that would be a good fit) (props [@oscarssanchez](https://github.com/oscarssanchez), [@barryceelen](https://github.com/barryceelen), [@felipeelia](https://github.com/felipeelia) via [#183](https://github.com/globeandmail/sophi-for-wordpress/pull/183)).
```

Comment:
Hi @jeffpaul the filter is actuallly `sophi_curated_post_list` https://github.com/globeandmail/sophi-for-wordpress/blob/9f768bf5341cacf97ccf9ba5813ed61138b88246/includes/classes/SiteAutomation/Integration.php#L113

A dev would not really be injecting posts into the Sophi response, but rather inject them previous to returning the filtered value from `posts_pre_query` https://github.com/globeandmail/sophi-for-wordpress/blob/9f768bf5341cacf97ccf9ba5813ed61138b88246/includes/classes/SiteAutomation/Integration.php#L31